src/tss_callbacks.c

```diff
@@ -0,0 +1,118 @@
+/*
+ * This file and its contents are licensed under the Apache License 2.0.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-APACHE for a copy of the license.
+ */
+
+#include <postgres.h>
+
+#include <fmgr.h>
+#include <executor/instrument.h>
+
+#include "guc.h"
+#include "tss_callbacks.h"
+
+#define TSS_CALLBACKS_VAR_NAME "tss_callbacks"
+#define TSS_CALLBACKS_VERSION 1
```

Comment:
Is this the required version of the data structure? Is the intention to be compatible with one version only or should it be backward compatible? Maybe we should add the term `MIN_VERSION` to the name and replace the `==` checks with `>=`.

Reply:
It is not clear to me yet how it will work when it change in future... so maybe would be better to change it in the future when the real problem raise